operator/internal/manifests/openshift/dashboards.go

```diff
@@ -0,0 +1,88 @@
+package openshift
+
+import (
+	"encoding/json"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+
+	"github.com/grafana/loki/operator/internal/manifests/openshift/internal/dashboards"
```

Comment:
Do we really need the extra nesting `internal/dashboards` instead of just `dashboards`?

Reply:
It became a habit in the code base to keep anything file/template handling in the internal package (see `loki-config.yaml`, etc.)